HotelBooking/HotelBooking.App/Operations/EmailSender.cs

```diff
@@ -0,0 +1,26 @@
+﻿using HotelBooking.Domain.Models;
+using HotelBooking.App.Operations.ExternalSystems;
+using System;
+using HotelBooking.Domain.Enums;
+
+namespace HotelBooking.App.Operations
+{
+    public class EmailSender : OperationBase
+    {
+        private IExternalEmailSystem _emailSender;
+
+        public EmailSender(IExternalEmailSystem emailSender)
+        {
+            _emailSender = emailSender;
+        }
+
+        public override bool IsRequiredToSucceed => false;

```

Comment:
W zadaniu to bardziej konfiguracja, caly algorytm procesowania mial decydowac czy operacja jest required czy nie, niz sama operacja.  To takze sprawa koncepcyjna